### PR TITLE
Add logging to output document processing

### PR DESCRIPTION
**Pull request by gjvis**
This adds logging to the output document processing without refactoring or rewriting how the processing itself works.

I'm not super happy with it, but it is the least amount of change to get logging in. I think this area needs attention, after which the logging will (along with everything else) hopefully end up cleaner.

### CODE REVIEW

File: app/services/upload_to_print_house.rb [hunk 1]
```diff
@@ -15,10 +16,27 @@ def call
...
-    Net::SFTP.start(ENV['SFTP_HOST'], ENV['SFTP_USER'], password: ENV['SFTP_PASSWORD']) do |sftp|
+    Net::SFTP.start(host, user, password: password) do |sftp|
+      logger.info("Connected to #{host}")
+
       sftp.upload!(io, path)
+      logger.info("#{path} uploaded")
     end
   end
+
+  def host
```
Comment (stevenwilkin) [inline app/services/upload_to_print_house.rb]: Moving the various params into methods would be best in a separate commit

File: app/services/create_batch.rb [hunk 2]
```diff
@@ -1,9 +1,19 @@
 class CreateBatch
+  def initialize(logger = Rails.logger)
+    @logger = logger
+  end
+
   def call
     unprocessed_appointment_summaries = AppointmentSummary.unprocessed.where(format_preference: :standard)
 
+    logger.info("Unprocessed appointment summaries: #{batch.appointment_summaries.count}")
```
Comment (andrewgarner) [inline app/services/create_batch.rb]: Where does `batch` come from?

File: app/services/upload_to_print_house.rb (hunk 1 above)
Comment (gjvis) [inline app/services/upload_to_print_house.rb]: Good point.

File: app/services/create_batch.rb [hunk 3]
```diff
@@ -1,9 +1,19 @@
 class CreateBatch
+  def initialize(logger = Rails.logger)
+    @logger = logger
+  end
+
   def call
     unprocessed_appointment_summaries = AppointmentSummary.unprocessed.where(format_preference: :standard)
```
Comment (stevenwilkin) [inline app/services/create_batch.rb]: Maybe a logging statement to lets us know this has been kicked off?

File: app/services/create_batch.rb (hunk 2 above)
Comment (gjvis) [inline app/services/create_batch.rb]: Last-minute-change oversight :(

File: app/services/process_output_documents.rb [hunk 4]
```diff
@@ -1,6 +1,15 @@
 class ProcessOutputDocuments
   def call
-    batch = CreateBatch.new.call
+    tags = []
+    tags << 'ProcessOutputDocuments'
+    tags << SecureRandom.uuid
```
Comment (stevenwilkin) [inline app/services/process_output_documents.rb]:
```ruby
tags = ['ProcessOutputDocuments', SecureRandom.uuid]
```
?

Comment (andrewgarner) [inline app/services/process_output_documents.rb]: What's the intent behind `SecureRandom.uuid`? If it is to signify a unique invocation of this method?

Comment (gjvis) [inline app/services/process_output_documents.rb]: I'll extract this into another class, and make sure the UUID is named.

File: app/services/process_output_documents.rb [hunk 5]
```diff
@@ -10,6 +19,22 @@ def call
 
     csv = CSVRenderer.new(output_documents).render
 
-    UploadToPrintHouse.new(csv).call
+    UploadToPrintHouse.new(csv, logger).call
+  end
+
+  def call_logged(logger)
```
Comment (stevenwilkin) [inline app/services/process_output_documents.rb]: bad taste in the mouth

**Comment by gjvis** [comment]
superceded by #154
